rel-eng/custom/custom.py

```diff
@@ -83,17 +83,10 @@ def filter_archives(path):
             new_line = "Source%s: %s\n" % (i, base_name)
             replacements.append((source_regex, new_line))
 
-        # Replace version in spec:
-        version_regex = re.compile("^(version:\s*)(.+)$", re.IGNORECASE)
-        self.version = self._get_version()
-        print("Building version: %s" % self.version)
-        replacements.append((version_regex, "Version: %s\n" % self.version))
-        self.replace_in_spec(replacements)
```

Comment:
This was trying to modify the version in place but I couldn't figure out why. It broke RPM building by incorrectly setting version. As far as I can tell, our specs are smart enough to not need this.

Reply:
I actually think this could be a good thing to have. If `VERSION` is bumped in git but we forget it here then they won't match. We probably need to drop `-develop` though.

Reply:
I wonder if we should change this code to fail if they don't match. Now it will fail during the build but the earlier you fail, the better.

Reply:
That feels like weird coupling to me to have this custom source strategy doing a validation check when we have builds that will do it or can put it into our pipelines.